### DISCUSSION TITLE
Not able to stop quartz logs

### DISCUSSION THREAD

**Discussion by ganupali** [quartznet/quartznet#1310] [Q&A]
Hi all 

I am facing one issue in my maven project where i am trying to stop quartz logs 

[org.springframework.scheduling.quartz.SchedulerFactoryBean#0_Worker-5] DEBUG o.s.jdbc.

I have tried the below options from stack overflow 
Jar files 
slf4j-api-1.7.30.jar 
log4j-frmc-1.2.17.3.jar

log4j.logger.org.quartz=OFF
log4j.logger.org.spring.jdbc=OFF
log4j.logger.org.springframework.jdbc.core = INFO

And in the JAVA class, I used the below code.

org.slf4j.Logger logg = LoggerFactory.getLogger(Class.forName("org.quartz.core.QuartzSchedulerThread"));
            if (logg != null && logg instanceof ch.qos.logback.classic.Logger)
            {
                //the slf4j Logger interface doesn't expose any configuration API's, but
                //we can cast to a class that does; so cast it and disable the logger
                ((ch.qos.logback.classic.Logger) logg).setLevel(ch.qos.logback.classic.Level.OFF);
               
            }

But it's not working after doing all the above changes still I am getting quartz logs in console. can any one please help me in this Thanks in advance.

**Comment by lahma** [comment]
Sorry but this repository is for the .NET version of Quartz, I'm afraid that I'm not able to help you here as my Java/Spring is quite rusty...
